domains/raghavyuva.json

```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "CNAME": "185.199.108.153"
+        "CNAME": "https://raghavyuva.github.io/portfolio/"
```

Comment:
Sorry, `CNAME` records cannot contain `https://` or `/`. Please change this to just `raghavyuva.github.io`

Reply:
changed